milestones/exceptions.py

```diff
@@ -0,0 +1,46 @@
+"""
+Application-specific exception classes used throughout the implementation
+"""
+from django.core.exceptions import ValidationError
+
+
+class InvalidCourseKeyException(ValidationError):
+    """
+    CourseKey validation exception class
+    """
+    pass
```

Comment:
Nitpick: Just as a note, if you have a docstring, you don't need to do a `pass`. So this would work:

```python
class InvalidCourseKeyException(ValidationError):
    """CourseKey validation exception class"""
```